src/web/core/src/main/java/org/geoserver/web/admin/ServerAdminPage.java

```diff
@@ -31,12 +31,7 @@ public GeoServer load() {
     }
 
     public IModel<GeoServerInfo> getGlobalInfoModel() {
-        return new LoadableDetachableModel<GeoServerInfo>() {
```

Comment:
This was the source of the glitches, ajax calls made the loadable detachable model reload the config object and changes would go bye-bye.

Reply:
reviewed